products/urls.py

```diff
@@ -9,4 +9,9 @@
 router = DefaultRouter()
 router.register(r"", views.ProductViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [
+    path("homepage/", views.ProductHomePageView.as_view(), name="homepage"),
```

Comment:
Can we show all products on `base/products` instead of `base/products/homepage`?

Reply:
Yes sure. Where should i move the api of products? i have moved it to `/products/api/` let me know if i should change that

Reply:
that's fine with me.